python/postprocessing/framework/postprocessor.py

```diff
@@ -238,7 +238,7 @@ def run(self):
                     eventRange=eventRange, maxEvents=self.maxEntries
                 )
                 print('Processed %d preselected entries from %s (%s entries). Finally selected %d entries' % (nall, fname, nEntries, npass))
-            else:
+            elif outTree!=None:
```

Comment:
`elif outTree is not None:`? :-)

Reply:
Done. The rest rest of the code indeed uses `is`.